ImageNet/train/huaweicloud_resnet50_modelarts_16_machine_tensorflow18.json

```
@@ -0,0 +1,38 @@
+{
+    "version": "v1.0",
+    "author": "ModelArts Service of Huawei Cloud",
+    "authorEmail": "zhangzhenyu6@huawei.com",
+    "framework": "TensorFlow v1.8.0",
+    "codeURL": "https://github.com/huawei-clouds/modelarts-example/blob/master/ImageNet-10min/fast_imagenet_dym_node_16.py",
+    "model": "ResNet-50",
+    "hardware": "modelarts",
+    "timestamp": "2018-11-22",
+    "misc": {
+        "model implementation": "https://github.com/tensorflow/models/blob/master/research/slim/nets/resnet_v1.py",
+        "backend": "8*Tesla-V100 + CUDA-9.0 + CuDNN-7.0.5",
```

Please split this into separate fields:

```suggestion
        "backend": "16 nodes (8*V100 for each node)",
        "CUDA version": "9.0",
        "CuDNN version": "7.0.2"
```